modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+          secret:
+            secretName: sgw-config
+----
+<1> `metadata.name`: The name of the deployment is "sync-gateway".
+<1> `spec.replicas`: 2 Sync Gateway replicas that are deployed at most.
+* For import node deployment, this *must* be 1 because there can be at most 1 Sync Gateway replicator in a cluster.
+If high availability is important, you probably want at least 2 Sync Gateway import nodes.
+* For replicator node deployment, this *must* be 1 because there can be at most 1 Sync Gateway import node in a cluster.
+<1> `containers[].image`: Points to the docker image for Sync Gateway.
+<1> `containers[].args`: Points to the Sync Gateway configuration file named "sgw-config-working.json" which is mounted at the path specified via the `volumeMounts` config.
+<1> `volumeMounts`: Specifies where to mount the volume into the container.
+<1> `volumes`: Specifies what to mount.
+In our case, the "secret" with name "sgw-config" corresponding to the Sync Gateway configuration that was created in the previous step is mounted.
+Learn more about Kubernetes volumes https://kubernetes.io/docs/concepts/storage/volumes/[here].
+. Deploy the Sync Gateway cluster using the deployment controller file.
++
+[source,console]
+----
+kubectl create -f sgw-deployment.yaml
+----
+If successful, you will see the following.
++
+[source,console]
+----
+deployment.extensions "sync-gateway" created
+----
+. You can check the status of the deployment with the following command until all the pods corresponding to the Sync Gateway are in the "Ready" state and the status is "Running".
++
+[source,console]
+----
+kubectl get pods
+----
++
+If successful, you will see the following.
++
+[source,console]
+----
+NAME                                 READY     STATUS    RESTARTS   AGE
+cb-example-0000                      1/1       Running   0          3d
+cb-example-0001                      1/1       Running   0          3d
+cb-example-0002                      1/1       Running   0          3d
+couchbase-operator-fd8db588b-9fzsw   1/1       Running   1          3d
+sync-gateway-7474f5df4b-c29xw        1/1       Running   2          18m
+sync-gateway-7474f5df4b-p98sq        1/1       Running   0          18m
+----
+
+== Deploying a Load Balancer
+
+In a production deployment, you will likely have one or more Sync Gateway nodes fronted by a xref:load-balancer.adoc[load balancer].
+
+You will deploy the load balancer using the https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/[Kubernetes Load Balancer service].
+The load balancer service provides an externally accessible IP address and routes traffic to the right ports in the cluster.
+
+Follow these steps to deploy a load balancer in front of the Sync Gateway cluster.
+
+. Create a new file called *sgw-load-balancer.yaml* with the following.
++
+[source,yaml]
+----
+kind: Service
+apiVersion: v1
+metadata:
+  name: sgw-load-balancer <1>
+spec:
+  selector:
+    app: sync-gateway <2>
```

Comment:
As above with copy+pastable-ness.

Reply:
@jamiltz ^^^ to get rid of the inline reference tags